addons/l10n_dk_nemhandel/tools/private_key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
mhhh not sure we want those informations ? Or are they only for testing purposes ? same for the others files of the tools  directory

Reply:
Removed most of them (as I don't feel like the demo bill was any value (and I needed to prepare a bill for it :smile: ).
And the .pem is pretty useless, I just left it, but it's only fake info

Reply:
if it's useless then why keep it ? Needed for testing purposes ?